index.html

```diff
@@ -4292,6 +4306,493 @@ <h3>Proof Formats</h3>
+<var>inputBytes</var> as an input parameter, parses the value as JSON into
+an object, removes the `proof` property, and returns the result as an object.
+If JSON parsing fails, a <a href="#PARSING_ERROR">PARSING_ERROR</a> is appended
+to <var>securingMechanismVerified</var>.<var>errors</var>.
+              </li>
+              <li>
+If the <var>inputMediaType</var> is associated with the JOSE or COSE
+specifications, the [[?VC-JOSE-COSE]] specification defines a process for
+extracting the <a>conforming document</a>.
+<div class="issue" title="Document extraction process in vc-jose-cose">
+The process for extracting a conforming document is not clearly documented in
+the [[?VC-JOSE-COSE]] specification yet, but once it is, a link to that section
+of the specification will be included in this text.
+</div>
+              </li>
+              <li>
+Other extracting functions are possible, but need not be listed in
+the <a data-cite="?VC-SPECS#securing-mechanisms">Securing Mechanisms</a>
+section of the Verifiable Credentials Specifications Directory [[?VC-SPECS]].
+Implementers might find these other specifications through other standards
+bodies or communities that utilize this specification.
+              </li>
+            </ol>
+          </li>
+          <li>
+Determine the specific cryptographic verification algorithm
+(<var>verifyProof</var>) required to verify the securing mechanism:
+            <ol class="algorithm">
+              <li>
+If the <var>inputMediaType</var> is `application/vc+ld+json` or
+`application/vp+ld+json`, the Verifiable Credential Data Integrity
+Specification [[?VC-DATA-INTEGRITY]] defines the
+<a data-cite="VC-DATA-INTEGRITY#verify-proof">process for identifying
+a specific cryptography suite type</a> used for cryptographic verification by
+utilizing the `proof`.`type` property. A list of registered proof types can be
+found in the
+<a data-cite="?VC-SPECS#securing-mechanisms">Securing Mechanisms</a>
+section of the Verifiable Credentials Specifications Directory [[?VC-SPECS]].
+                <ol class="algorithm">
+                  <li>
+Set <var>securedDocument</var> to the result of a JSON parsing function that
+takes <var>inputBytes</var> as an input parameter, parses the value as JSON,
+and returns the result as an object. If JSON parsing fails,
+<var>securingMechanismVerified</var>.<var>status</var> is set to `false`, and a
+<a href="#PARSING_ERROR">PARSING_ERROR</a> is appended to
+<var>securingMechanismVerified</var>.<var>errors</var>.
+                  </li>
+                  <li>
+If <var>securedDocument</var> was successfully set in the previous step, set
+<var>securingMechanismVerified</var>.<var>status</var> to the result of passing
+<var>securedDocument</var>, along with any relevant function-specific options,
+to the <var>verifyProof</var> function. For example, the
+<var>expectedProofPurpose</var> option is set to `assertionMethod` when
```

Comment:
proof purpose is again data integrity proof specific, it also shows up in controller documents... this is creating normative spaghetti monster : )

Hopefully we can rely on citations informatively, instead of adding these details normatively.

https://w3c.github.io/vc-controller-document/#assertion

Reply:
> Hopefully we can rely on citations informatively, instead of adding these details normatively.

We can.

> It seems like this logic should be in data integrity, and not needed in the core data model.

What logic? The text you're highlighting starts with "For example", because it's an example. The text is there because there was confusion (over the past several weeks of discussion with other members of the WG) over what sorts of options might be send to a securing mechanism verification function.

Reply:
commented on securing mechanism specific optionality in the core data model is a layering violation.